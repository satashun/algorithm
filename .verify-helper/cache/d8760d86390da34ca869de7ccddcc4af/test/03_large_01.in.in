1000000000110010000010100000000001000000010101001001100000001010110010010101010000011000000000000000100011010000000000000010001000011000000000010001000000001001000001000000000000001000001000010000001000100000001000110000100100000000111001011000100101100010001100000000100000001000000000000000000010011000100000100001110010000001001100000000100000001011001100001000001001000100010010000000001100011000000100100000001011100110001000000010000011000100100000010010001000000000101100100010000100010000010000001000001100010000010000000010001111001101000010000100100100100000010000000001000000000000000000000000010000010011001010000110100000010000101000100010100000000011010001010010000000000000001101010010000110100011000001100001000000010100101000001000000000000001100001001000000001000000000000100001000000010000000000011010000000100110010000000000000001000010000000000000100000000100000000011000010000000000100000000011000001000010011101000000000000100111000001000001000010000000000100000100100000100100010010000001101000000001000101000000000000000000000000000000000000000000000000000000000001000000000000000000000000010000000000100010000000000000000000001000000000001110001000100000000000000000000000001000000000100000000000000001010000010010000000000000000000000000000000000000000000000010000000000001010000000000100000000001000000000000000010000000100000000000000000100011000000000000000110000000100000000000100000000100000000000000100010000100000100000000000000000000000000000000000000000000000000001000000000100000000000000010010000000000000000000000000000000000000000000000000000000000000000100000000010000000000000000000100100000000000000000000001001000010001000010000010000000000000000000000000000000010100010000000000001000000000000010000000000000000000000000000000000000010000000000010100000000000000100010000010000000000000000000000001000000010000000010100001000000001000000000000010000000000000000010000000000000000000000010000000000001000000110100000100000000001000001000000000000000000000000000000000000000000000000000001000000000001000001000000000000000000000101000000010100000100000000000001000000000000000000000000000000000100001000000000100100000000000000000000000000000100010000100010000100000000000000000000000000000000000001000000100000000000000000010000001010000000000000000000000000000000010000000000000000100000000000000001000011001000000100000001000000001000000000000000100110000000000000000100000010001000000001010000000000000000000000010000000000001000000000000000000100000100000000000100000010000000000000000000000000000000000000000100000000001000000000000000000000000000001000000000000010010000000001000000001000000000000000000000000000110000000000000000000000000010000000000000000001001000001000000000000000010000000010000000000000000000001000000000000010100000000000000000001000100000000100110000000000000000000000001000000000000100110000010000000010000000000000000100000000000000000000000000000000000000100000100000000000000000000011001000100000000010001000000000000000000000000000000000000000000110000000010000000000000000000000000000000010000010000000001000000000000000000000000000100000000001000000000000000000010000000000000000000000000000000100000000000000100001000000010000101001000000000000000000000000100000000000101110000000010001000000101001000000000000000000000000000000000000000110000000000010000000000000000000000000000000000010000000000000000000000000001000000000100000000001000000000000001000000001000000000000000000000000000000000000000000000100000010000000000000100000000000000000000000000000001000000000000010000000000000100000000000000000000000000100001001001000000000000000000001000000000000010000000000001000000000001000010001000000000000000000010000000000000100000000100000000001000010100000000100000000000000000000000100100000000000000000000000000000000000000000000000000000000000000001000010000000000000000000000000000000010100001000000000100000100000000000000000000000000000000000000000000000000000100010000010100000000000110100100000000000000000000000000000010000100100000000000000000000000000010000001000000100000010100000000000010000010000000000000000001100000000100000001000100000000000000000000100000000000000000000001000000000000000001000000101000000000010000000000000000100000000000000001000000000000000000000000100000100000000100000000000000000001000000000000000000000010000000100000100000000000000100000001000000000000000000000000000100000000000000000000000000100000000000000000001001001001001000000000000000000000000010000000100100000000000000000000000000000000000000000000000000000000000000000001000000000000000010000100000010000000000010000100100000000000000000000010000000000000001101110000000000000000000000000001000010000100000100000001000101000000000000000000000000000001000011010000001000000000100000000000000100000000000000000010000000000000000000000001000000000000101000000010000000100010000000000010000000000000000000001100001000000000000000000011000000000001000000000100000000000000001000001000001000001000001000000000000100000001000000000001000000000100000000010000000000000000000010000000000000101000000000000100010000000000000100000000100000000000000100000100000000000000000000000000000000000100010010000001100000000000000000000000000000001100000101000000000100000010010000000000000000000000010000010010001000000000001000000001000000000001000000100000000000010000000000000000100000000000100000000100000000000000000010000010000000110010000100000000000000001000000010100010000000000000000000000000000100000000000000000000001000000000000000001010010000000000000000000000100000100000000000000000000011000000000000000000000010100000000000000000000000010000000101000000001000100000000000000010000000100001000010000000100000000000010000110000000000000000000000000000000000001010000001000000000000000000100000000000010100001000100000000000000000000000000000000000000000010010000000100110100000000000000000000000001100000000010000000000010100010000000010001000000000000000010001000000100000000100000000000000000000001000000000000000001100010000000000000000000000000000000000000000000000010000000000001000000000000000000000000000000000000000010000000000000000001000000000100010000000000000000000000000000000000010000100000100000000000100010000000000100000000000000000000011000100000000010000000001000000000010000000000000000000000000001110000000000000000000000010000000000000010000000000000000001000000000000001000000001000000000000000000000000001000000000000000000000000100000000001000000000000000000000000000000000010010000101000001010000000000100000000000000100000000000000000001001000000000000001000000000000100000000100001000000000000000100000010000000000000000000000000000000000000000010000000000000000000000000000000010000000000100011000000000000000000001000000000100000000000010000000000000000000000000000000101000000000000010000000000000000000000000000010100000000000000001010010000000000000000001000000000000000000000000000000001000010001000000000000000000000000000100000000000000000000000000010000000000000000000001000000000000000000000000001000010010000000000000100100100010000000000000000000000000000000000000000010000000000000000100000000000001100000000000000000000000010000000001100000100001000010000000010000000000000000110000000000010000000000000001000000000000010100000000000000000000000000000000000000000100000000000000000010000000000000010000000001100010000001000100000100000000000000001011100000000010000000000001000000000000000000000001000100000000000000000000000000000000000000000000110000000000000000000000100000000100000000000000100000010000100000000000000000000000101000000110000000000001000000000000000000000100000000001000001000000000000000000000000000010000000000000110000100000000010000000000000011000000000000000000100000000000010000010000000000000001001000000010010000000000000000100000000100000000010000000000000001100000100000000001000000000000000001000000000010000000000000000000000000100101100010000000000000000000001000000000001000000000011000000000000000010000000000000000001000000000000100000000000110000010000000001000000000000000000100000000000001000010000010000000000000000000000000000000010000100000000000010000000000010000000000000100000000001000000000101000001000100001000000010000010000000100000000000010000000000010000000000001110000000000000000001000010100000000000000000000000000000000100100000100000010000000000000000001000000000000000000000000010000000010000000100000000000000000000000000000000000000010000000000000001001000000001000000000010000000001000010000010000000000000000000000000000000000000000000000000000000000001000010000011010100001000000000000000001000000000000000100010000001000000000000000000000000000000000000100000000000010000100000001000000000000000000000000000000000000000000001010000000000000000000100100000011000000000001000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000100100000100000000000000000000000000010000000000001001000000000000011000010001000000100100000100000000000000000000000000000010000100000100000000000010100000100000010000000000000000000000010000000000000000010000000010001000100010000100000000000001101101000000000001000010000000000000000100100000000000000000000101001000000110000000000000000000010000000001001000000001000000001000000001010000000000000000000110000000000001000000011000100000100000000010000001000000000001000100000001100000000000101000000000000000010000001000000000100000000000000000000000000000000000000100000000000000000000001000010101000000100010000000000000000000000000000100100000000000000001000000100000000010000000000000000101000000100000000000000000000100101000000000000000000000011010000000000000000001001000000000000000000000000001000000000001000000000000000000000000000010000000101100100000000000100000000010000000000000000000100010000000000000000000000000000000000011000000000000000000000000000000101000000000000000000100000000000000100010000000000000100000000000111000000000000100100001000000010000001000010000000000100000000000000011
000000000011
