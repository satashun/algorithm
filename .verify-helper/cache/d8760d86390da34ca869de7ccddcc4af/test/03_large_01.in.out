680
749
805
1154
1377
2652
2950
3015
3309
3966
4100
4637
4883
4909
5207
5563
5873
5993
6229
6293
7115
7194
7464
7653
7684
7805
7933
7995
8231
8947
9125
9285
9631
9820
9918
9988
